pytorch/buildspec.yml

```diff
@@ -1,7 +1,7 @@
   account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
lines 79 - 103 specify the build for inference images, these may need to be commented out

Reply:
already commented. thanks!